communication/README.md

```diff
@@ -146,22 +146,21 @@ Office hours are held once a month. Please refer to [the docs][office-hours]
 to learn more. Join the conversation on slack under [#office-hours].
 
 ### Other
-- [r/kubernetes] - reddit channel owned and operated by community members and
-not an official channel for the project.
+- [r/kubernetes] - reddit channel owned and operated by community members and not an official channel for the project.
```

Comment:
qq - what is the change here? :x
If it removing the additional line, the original form conforms with our style guide at keeping them at 80~ characters.

Reply:
replaced with original one, please review it